README.adoc

```diff
@@ -21,16 +21,29 @@ up an issue here on Github or by contacting @cwinsnes.
 
 == Installation and Setup
 NOTE: It is recommended to run the module in a separate virtual environment
-such as https://www.anaconda.com/[Anaconda] or
+such as https://conda.io/projects/conda/en/latest/user-guide/install/linux.html[miniconda] or
```

Comment:
I believe this should link to the Miniconda page rather than the linux install.

 https://docs.anaconda.com/free/miniconda/